package.json

```
@@ -14,7 +14,7 @@
 		"cross-seed": "dist/cmd.js"
 	},
 	"engines": {
-		"node": ">=16"
+		"node": ">=18.18.2"
```

```suggestion
		"node": ">=18"
```

Same reasons as above, I would set this to `>=18`